oozie-to-airflow/converter/oozie_converter.py

```diff
@@ -158,7 +158,13 @@ def write_nodes(self, file: TextIO, nodes: Dict[str, ParsedNode], indent: int =
         :param indent: integer of how many spaces to indent entire operator
         """
         for node in nodes.values():
-            file.write(textwrap.indent(node.mapper.convert_to_text(), indent * " "))
+            tasks, relations = node.mapper.convert_tasks_and_relations()
```

Comment:
As we discussed offline @mik-laj, I'd suggest changing the name of `convert_tasks_and_relations()` to something like:
* `create_tasks_and_relations()`
* `generate_tasks_and_relations()`
* `get_tasks_and_relations()`
* `to_tasks_and_relations()`

Reply:
I updated the PR. I with @potiuk chose last variant. Thanks for suggestion.

Reply:
:+1: